src/polus/plugins/__init__.py

```diff
@@ -6,6 +6,7 @@
     get_plugin,
     list_plugins,
     load_plugin,
+    print_invalid,
```

Comment:
see comments below. We may not want to add this function to the api.

Reply:
This is no longer part of the api